### PR TITLE
Configure CI SN-690

### PR DESCRIPTION
**Pull request by vasiliy-chefonov**
I created a CI workflow that automatically packs the library and releases the .nupkg file. This way you can use it for local testing without having to package the project on your side manually.

- [VS instruction](https://stackoverflow.com/questions/10240029/how-do-i-install-a-nuget-package-nupkg-file-locally-to-visual-studio)
- [Rider instruction](https://rider-support.jetbrains.com/hc/en-us/community/posts/4411200978834-Can-I-add-packages-outside-of-NuGet)

 
**Workflow:**

- Reads the version from the VERSION.txt file.
- Appends a version_postfix depending on the environment.
- Updates the version in .csproj and .nuspec files.
- Restores packages and builds the solution.
- Packs the library into a .nupkg file.
- Creates a GitHub release with the new version.
- Uploads the .nupkg file as a release asset.

**Dev Notes:**
- Do not forget to update the [VERSION.txt](https://github.com/saritasa-nest/saritasa-forge-admin/blob/develop/VERSION.txt) each time you merge your feature. Refer to [Semantic Versioning](https://semver.org/).
- Please, update the [CHANGELOG.txt](https://github.com/saritasa-nest/saritasa-forge-admin/blob/develop/CHANGELOG.txt) file with any changes you applied to track them.

### CODE REVIEW

File: demo/Saritasa.NetForge.Demo/Saritasa.NetForge.Demo.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
         <TargetFramework>net8.0</TargetFramework>
         <Nullable>enable</Nullable>
         <ImplicitUsings>enable</ImplicitUsings>
+        <Version>0.1.0</Version>
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo/Saritasa.NetForge.Demo.csproj:7]: Versions in `.csproj` files we also need to update everytime?

**Review by AlexanderMakarov124 — APPROVED** [review]
Can we add workflow for checking changes in `CHANGELOG.txt` and `VERSION.txt` so we restrict merge to `develop` if there are no changes in these files?

File: demo/Saritasa.NetForge.Demo/Saritasa.NetForge.Demo.csproj (hunk 1 above)
Comment (vasiliy-chefonov) [reply demo/Saritasa.NetForge.Demo/Saritasa.NetForge.Demo.csproj:7]: No, it will be automatically updated during the build.

**Comment by vasiliy-chefonov** [comment]
> Can we add workflow for checking changes in `CHANGELOG.txt` and `VERSION.txt` so we restrict merge to `develop` if there are no changes in these files?

Sometimes, changes are related to the doc updates or demo db schema. So, I'm not sure we need to release the new version in this case.

**Pull request merged**